### ISSUE TITLE
how to generate the image feature file?

### ISSUE THREAD

**Issue by wjy3326** [ChenRocks/UNITER#41]
I cannot find the code tools/generate_npz.py,  i don't know how to generate the image feature file. Could you  please upload your code? Thanks!

**Comment by RalphHan** [comment]
The same issue. Could you please upload your code? Thanks!

**Comment by abhinavm0** [comment]
+1

**Comment by floschne** [comment]
+1

**Comment by floschne** [comment]
@ChenRocks Can you please explain how the image features are created? In the paper, you state that they're from Faster RCNN with ResNet 101 (2048 dim) and the 7-dim position vector but how exactly are they processed before stored in the lmdb?

**Comment by floschne** [comment]
@linjieli222

**Comment by floschne** [comment]
@RalphHan @abhinavm0 @wjy3326 checkout this issue: https://github.com/ChenRocks/UNITER/issues/18

**Comment by abhinavm0** [comment]
Thanks @floschne !!
What a coincidence... just saw this yest morning myself... it works from within the docker though with Py27 running.. somehow not able to install caffe on bare EC2.

**Comment by floschne** [comment]
yeah I also had a lot of troubles to get caffe running on my machine (multi CUDA, cuDNN ... welcome to the config hell)
What I did, in the end, was getting lambda labs instance and it worked out of the box like a charm!

**Comment by onedayseek** [comment]
Hi! I'm trying to reproduce the UNITER-base model on the RefCOCO dataset. Unfortunately, the official download script no longer works—Azure has shut down public access, so I can't get the preprocessed features.

Due to limited compute resources, I really need the pre-extracted features to proceed (features for other datasets are also welcome). If you still have a copy and would be willing to share, I'd be incredibly grateful. Thanks in advance! 🙏
